docs/gambit/mutation-verifier.md

```diff
@@ -0,0 +1,99 @@
+# Using Gambit with the Prover
+
+This is the mutation verifier which
+ checks that variants of the original
+ solidity program do not pass the specification.
+It uses mutations from the [Gambit](https://github.com/Certora/gambit)
+  mutation generator.
+It also allows users to include manually generated mutants.
+If a mutated program passes the specification,
+it may indicate that the specification is vacuous or not rigorous enough.
+In the rest of the document,
+  we refer to both the mutation generator and the verifier as Gambit.
+
+## Installations and Setup
+
+You will require java to run the mutation testing jar.
+Installation instructions can be found [here](https://www.java.com/en/download/help/download_options.html).
+
+## Running the Mutation Verifier
+
+- Example:
+```
+java -ea -jar $CERTORA/certora_jars/MutationTest.jar /path/to/config/file/Example.conf
+```
+
+- **NOTE: if a path has spaces, put quotes around it to ensure correct parsing by Kotlin's libraries**
+
+- Gambit supports `--staging`!
+
+**However, Gambit currently has trouble with
+`--send_only` and `--cloud` in the run scripts.
+If you have these flags, please remove them for now.**
+Apologies for the temporary inconvenience!
+
+The tool expects a configuration file (extension `.conf` is required)
+which is a JSON object.
+Here is an example configuration file:
+
+```
+{
+    "project_folder" : "Test/10Power",
+    "run_script" : "Test/10Power/runDefault.sh",
+    "gambit" : {
+        "filename" : "Test/10Power/TenPower.sol",
+        "solc" : "solc8.10"
+    },
+    "staging" : "master",
+    "manual_mutations" : {
+        "path/to/original/file.sol" : "path/to/manual/mutations/",
+        ...
+    },
+    ...
+}
+```
+
+Required Keys for the JSON Configuration File:
+- `"project_directory"` : the directory containing the original CVT project on which to perform mutation testing
+- `"run_script"` : the bash script used to run verification on the original project, usually `project_directory/run.sh` or similar
+- `"gambit"` : the JSON configuration element for invoking Gambit. May be a path to a gambit configuration file
+or the explicit JSON element contained therein. Refer to the open source Gambit repository for more details regarding Gambit configuration files.
```

Comment:
```suggestion
or the explicit JSON element contained therein.  See {ref}`gambit-config` for more information about the gambit configuration.
```

Reply:
You'll have to add `(gambit-config)=` on the line before the corresponding section in the other document for the `ref` to work.